ja.po: ch_ops_log_monitor.xml

amotoki opened the pull request.

amotoki committed:
ja.po: ch_ops_log_monitor.xml

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_ops_log_monitor.xml:23
+msgid "Where Are the Logs?"
+msgstr "ログの位置について説明します。"
```
見出しとしては自然ではないと思います。
「ログはどこにあるのか？」くらいでしょうか。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"subdirectories of the <code>/var/log directory</code>."
+msgstr ""
+"Ubuntuでは、ほとんどのサービスでは<code>/var/log</code>ディレクトリ配下のログ"
+"ファイルを置いています。"
```
subdirectories of /var/log directory を訳し出した方がよいでしょう。

「Ubuntu では、ほとんどのサービスが /var/log ディレクトリ以下のディレクトリにログファイルを出力するという慣習に従っています。」

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_ops_log_monitor.xml:33
+msgid "Log Location"
+msgstr "ログの位置"
```
「ログの場所」？

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"次の通りです: DEBUG、INFO、AUDIT、WARNING、ERROR、CRTICAL、TRACE。特定のログ"
+"レベルより\"重要\"な場合のみメッセージはログ中に出力されます。DEBUGは全てのロ"
+"グが出力されます。例えば、TRACEはソフトウェアがスタックトレースを持つ場合にの"
+"みログに出力されます。一方INFOは情報のみのメッセージも含めて出力されます。"
```
日本語の方では、ログレベルが主語になっているのは、主語と述語が一致していません。

「ログレベル DEBUG の場合、すべてのログが出力されます。」
「例えば、TRACE では、」
「INFO では、～」

L.165 
「重要度順のレベル」だと、低い順か高い順か分からないと思います。推測は付きますが :-)
「レベルは次の通りです（重要度の低い順）：」

L.167
「ログ中に」→「ログに」

takekawa replied:
「ログレベルDEBUGの場合、」
「TRACEの場合、」
「INFOの場合、」
にしました

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+#: doc/src/docbkx/openstack-ops/src/ch_ops_log_monitor.xml:113
+msgid "To disable DEBUG-level logging, edit <code>/etc/nova/nova.conf</code>:"
+msgstr ""
+"DEBUG レベルを無効にするには <code>/etc/nova/nova.conf</code> を編集します。"
```
DEBUG レベルのロギングを無効にするには、以下のように nova.conf を編集します。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"この例において、cinder-volumesは起動に失敗し、スタックトレースを出力していま"
+"す。ボリュームのバックエンドがストレージボリュームとしてセットアップされてい"
+"ないためなので、おそらく、設定ファイルで指定されたLVMボリュームが存在しないと"
+"考えられます。"
```
「この例において」→「この例では」

「ボリュームのバックエンドがストレージボリュームをセットアップできないため、おそらく、設定ファイルで指定された LVM ボリュームが存在しないため、と考えられます。」
（unable なので、されていない、ではなく、できない、とすべきです。）

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"The typical way is to trace the UUID associated with an instance across the "
+"service logs."
+msgstr ""
+"一般的な方法はこれらのログインスタンスのUUIDをキーにして追跡することです。"
```
「インスタンスの UUID をキーにして、複数のサービスのログを追跡することです。」

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"log</code>で見つかります。同様にComputeノードで検索した場合、<code>nova-"
+"network.log</code> と <code>nova-compute.log</code>で見つかります。もし、"
+"ERRORやCRITICALのメッセージが存在しないならば、最も最後のログエントリが、何が"
+"悪いかのヒントを示しているかもしれません。"
```
「存在しないならば」→「存在しない場合」
「最も最後の」→「最後の」
くらいでよいと思います。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"file. For most files, these should already be there:"
+msgstr ""
+"ログステートメントを追加するには、次の行をファイルの戦闘に置きます。ほとんど"
+"のファイルでは、これらは既に存在します。"
```
戦闘→先頭

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"RabbitMQ Web 管理インターフェイスを有効にするもう一つの方法としては、"
+"<placeholder-1/> コマンドを利用します。例えば<placeholder-2/> は、キューに"
+"残っているメッセージを表示します。もしメッセージ存在する場合、Cinderサービス"
+"がRabbitMQに正しく接続できてない可能性があり、再起動が必要かもしれません。"
```
http://dream.daynight.jp/openstack/openstack-ops/content/logging_monitoring.html#rabbitmq では翻訳されないままになっています。理由を確認する必要がありそう。

takekawa replied:
元のXMLとmessageの内容が異なるようでした。修正します。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"queues and the processing time statistics for the server."
+msgstr ""
+"RabbitMQの監視すべき項目は、各キューでのアイテムの数と、サーバーでの処理時間"
+"の統計情報があります。"
```
「RabbiｔMQ で監視すべき項目としては、」

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+msgstr ""
+"クラウドは多くのサーバーから構成されるため、各サーバー上にあるイベントログを"
+"繋ぎあわせて、ログをチェックしなければなりません。よい方法は全てのサーバーを"
+"一ヶ所にまとめ、全てのログを同じ場所で確認できるようにすることです。"
```
「全てのサーバーを」→「全てのサーバーのログを」

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+msgstr ""
+"Ubuntuはrsyslog をデフォルトのロギングサービスとして利用します。他に何かをイ"
+"ンストールすことなく、設定ファイルを変更するだけで、リモートへログを送ること"
+"ができます。リモート転送を実施する際は、盗聴を防ぐためにログが自身の管理ネッ"
```
「rsyslog はもともとリモートにログを送信する機能を持っているので、何かを追加でインストールする必要はなく、設定ファイルを変更するだけです。」

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"the IP points to the Cloud Controller."
+msgstr ""
+"これは、rsyslogに全てのログを指定したIPに送るように命令しており、このIPはクラ"
+"ウドコントローラーを指定しています。"
```
指定しています→指しています

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"choose a server that is solely dedicated to this purpose. Create a file "
+"called /etc/rsyslog.d/server.conf with the following contents:"
+msgstr ""
+"ログサーバーを指定して下さい。ログ専用のサーバーを利用するのが最も良いです。"
```
ログサーバーを指定して下さい→集中ログサーバとして使用するサーバーを決めます。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"example.com goes to the following locations:"
+msgstr ""
+"この設定例はnovaサービスのみを扱っています。はじめにUDP 514番ポートを利用する"
+"rsyslogがサーバーとを設定します。次に一連のログテンプレートを作成します。ログ"
```
「はじめに rsyslog を UDP 514番ポートで動作するサーバとして設定します。」

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+msgstr ""
+"NagiosとNRPEを使って、クリティカルなプロセスの自動化されたアラートを作成する"
+"ことが可能です。<code>nova-compute</code> プロセスがコンピュートノードで動作"
+"していることを保証するために、次のようなアラートを作成します。"
```
「Nagios サーバーで」が抜けています。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+msgid ""
+"Then on the actual compute node, create the following NRPE configuration:"
+msgstr ""
+"そして、対象のコンピュートノードにおいて、次のようなNPRE設定を作成します。"
```
NPRE→NRPE

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+msgstr ""
+"リソースのアラートは、一つ以上のリソースが極めて少なくなった場合に通知するも"
+"のです。監視の閾値はインストールされたOpenStackの環境に合わせて設定すべきです"
+"が、リソース監視の利用方法は、OpenStackに固有の話ではなありません。一般的なア"
```
「では」と「ありません」の間の「な」が余計です。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"メモリ、ディスク、CPUのような一般的なリソースは、全てのサーバー(OpenStackに関"
+"連しないサーバーにも)に存在し、サーバーの状態監視に重要です。OpenStackの場"
+"合、これらのリソースは、インスタンスを起動するために十分利用可能であること保"
+"証するという点でも重要です。OpenStackのリソース利用を見るための幾つかの方法が"
```
「十分利用可能」というのが分かりにくかったです。「こと」の後ろに「を」が必要です。

インスタンスを起動するために十分なリソースがあることを保証するという点でも重要です。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+msgstr ""
+"リソース利用をテナントのクォータと結合することで、利用率を求めることが出来ま"
+"す。例えば、テナントが、10個のうち1つのFloating IPを利用しているとすると、10%"
+"のFloating クォータを利用することになります。これを行なうことで、レポートを作"
```
Floating → Floating IP

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"(https://github.com/cybera/novac/blob/dev/libexec/novac-quota-report)."
+msgstr ""
+"上記の出力はこのGithub (https://github.com/cybera/novac/blob/dev/libexec/"
+"novac-quota-report) 中のカスタムスクリプトを用いて作成しました。"
```
「中の」→「にある」

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"registry</code> processes are running or by seeing if <code>glace-api</code> "
+"is responding on port 9292."
+msgstr ""
+"インテリジェントなアラートは継続的インテグレーションの要素の一つです。\n"
```
「運用における継続的インテグレーション」
for operations が抜けています。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+msgstr ""
+"インテリジェントなアラートは、この章で述べられているの他のアラートよりも時間"
+"をかけて、計画、実装のために考えます。インテリジェントなアラートを実装するた"
+"めのよいアウトラインは次の通りです。"
```
「時間をかけて計画、実装のために考えます」→「計画、実装にかなり時間を要します。」

アウトライン→手順、流れ

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+msgstr ""
+"トレンドはどのようにクラウドが実行されているかの素晴しい見通しを与えてくれま"
+"す。例えば、負荷の高い日がめったに発生していかどうかや、新しいコンピュート"
+"ノードを追加すべきかどうか、などです。"
```
「負荷の高い日がたまたま発生したのか」

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+"reporting. Some other trend examples include:"
+msgstr ""
+"以前に示した全てのアラートタイプは、トレンドレポートに利用可能です。その他の"
+"トレンドの例は以下の通りです。"
```
「以前に示した」→「これまでに説明した」 （ここは時間的な話ではなく、上で述べたことを指しています）

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,964 @@
+msgstr ""
+"このコマンドを定期的に実行し結果を記録することで、トレンドレポートを作ること"
+"ができます。これにより<code>/var/log/nova/nova-api.log</code>の使用量が増えて"
+"いるのか、減っているのかをしっかり判断することができます。"
```
/var/log/nova/nova-api.log → nova-api

keeping steady が抜けています。
「増えているのか、減っているのか、安定しているのか、を知ることができます。」

takekawa committed:
Fixed Motoki-san comments
https://github.com/openstack-ja/openstack-manuals/pull/28

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,970 @@
+#: doc/src/docbkx/openstack-ops/src/ch_ops_log_monitor.xml:113
+msgid "To disable DEBUG-level logging, edit <code>/etc/nova/nova.conf</code>:"
+msgstr ""
+"DEBUG レベルのロギングを無効にするには、以下のように nova.conf を編集します。"
```
原文は code タグで囲まれているので /etc/nova/nova.conf になっているので、それに合わせる必要があります。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,970 @@
+"subdirectories of the <code>/var/log directory</code>."
+msgstr ""
+"Ubuntu では、ほとんどのサービスが /var/log ディレクトリ以下のディレクトリにロ"
+"グファイルを出力するという慣習に従っています。"
```
/var/log を code タグで囲む必要があります。

takekawa committed:
Added <code> tag to nova.conf file

takekawa committed:
Added <code> tag to /var/log

takekawa commented:
&lt;code&gt; タグの指摘を2箇所なおしました(ひとつ気づくのがおくれて二回コミットしてます)。
msgfmtの結果はいかのとおりです。

```
 (trans-ja/ch_ops_log_monitor.xml) $ msgfmt -o /dev/null --statistics ch_ops_log_monitor.xml.ja.po
135 個の翻訳メッセージ.
```

amotoki commented:
修正ありがとうございます。私のチェック分は完了にしたいと思います。

takekawa commented:
こちらこそありがとうございました。勉強になりました m(_ _)m

Commit:
Update the original string

The original document is updated and it needs to be used to generate
the translated version of the document. It is better to update ja.po
to catch up the latest.

Note that "latest" here is HEAD revision of master branch
(commit efe410a887bce532962c689be451e9b6d5875693).

Sync with new operation-guide repository is the future work.

amotoki commented:
openstack-ja repo の ops-guide で翻訳ドキュメントを生成する際に未翻訳で残ってしまうので、
原文が更新されている場所を更新しました。 placeholder タグへの置き換えです。
https://github.com/openstack-ja/openstack-manuals/commit/a7e49bc1aeefb6a07af6d9b01a8412935f4fba9e

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,969 @@
+"for debugging OpenStack related issues. Instead, we recommend you use the "
+"RabbitMQ web management interface. Enable it on your cloud controller:"
+msgstr ""
+"接続失敗の問題はさておき、RabbitMQログファイルはOpenStackに関連した問題のデ"
```
「接続失敗の問題は別とすると」でしょうか。一般的には役に立たないことを言っていますので。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,969 @@
+"- probably because the LVM volume that is expected from the configuration "
+"does not exist."
+msgstr ""
+"この例では、cinder-volumesは起動に失敗し、スタックトレースを出力していまボ"
```
「この例では、ボリュームのバックエンドがストレージボリュームをセットアップができなかったため、cinder-volumesが起動に失敗し、スタックトレースを出力しています。おそらく、設定ファイルで指定された LVM ボリュームが存在しないためと考えられます。」としてみました。

〜のための方は has been の意味を込めて「できなかったため」としています。一方、「スタックトレースを出力しています」の方は具体例がその場で見えているので、日本語としては「〜しています」の方が自然だと思ったので、そうしました。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/ch_ops_log_monitor.xml.ja.po:
```diff
@@ -0,0 +1,969 @@
+"Once enough points in time have been recorded, you can see how the value has "
+"changed over time."
+msgstr ""
+"トレンドはアラートとは全く異なったアプローチです。アラートは二値の結果(チェッ"
```
そうですね。二値の結果よりは分かりやすいですね。

amotoki committed:
Address comments from Kato-san
